From 57bd540be548e35931d842713f217903ec34c0ac Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Sun, 6 Mar 2016 11:33:39 +0200
Subject: [PATCH 0927/1240] dts: a70x0: Use a70x0-rz specific dtsi file for rz
 board setups

Use 70x0-rz.dtsi for rs stepping, and 70x0.dtsi for a0 stepping.

Change-Id: I94bfaf967e0d1f5a0a1eb5ffdb227daea1a022bc
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28054
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/dts/armada-7040-rz-db_all_pci.dts    |  2 +-
 arch/arm/dts/armada-7040-rz-db_dop.dts        |  2 +-
 arch/arm/dts/armada-7040-rz-db_kr.dts         |  2 +-
 arch/arm/dts/armada-7040-rz-db_nand.dts       |  2 +-
 arch/arm/dts/armada-7040-rz-db_rgmii_only.dts |  2 +-
 arch/arm/dts/armada-7040-rz-db_router.dts     |  2 +-
 arch/arm/dts/armada-7040-rz-db_rxaui.dts      |  2 +-
 arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts  |  2 +-
 arch/arm/dts/armada-70x0-rz.dtsi              | 24 ++++++++++++++++++++++++
 9 files changed, 32 insertions(+), 8 deletions(-)
 create mode 100644 arch/arm/dts/armada-70x0-rz.dtsi

diff --git a/arch/arm/dts/armada-7040-rz-db_all_pci.dts b/arch/arm/dts/armada-7040-rz-db_all_pci.dts
index c8865ae..df63140 100644
--- a/arch/arm/dts/armada-7040-rz-db_all_pci.dts
+++ b/arch/arm/dts/armada-7040-rz-db_all_pci.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_dop.dts b/arch/arm/dts/armada-7040-rz-db_dop.dts
index cbe4ece..4115c50 100644
--- a/arch/arm/dts/armada-7040-rz-db_dop.dts
+++ b/arch/arm/dts/armada-7040-rz-db_dop.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_kr.dts b/arch/arm/dts/armada-7040-rz-db_kr.dts
index 8e5a5f7..44ed45b 100644
--- a/arch/arm/dts/armada-7040-rz-db_kr.dts
+++ b/arch/arm/dts/armada-7040-rz-db_kr.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_nand.dts b/arch/arm/dts/armada-7040-rz-db_nand.dts
index 18bbdc7..05cf30b 100644
--- a/arch/arm/dts/armada-7040-rz-db_nand.dts
+++ b/arch/arm/dts/armada-7040-rz-db_nand.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts b/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts
index 20537eb..aca69af 100644
--- a/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts
+++ b/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_router.dts b/arch/arm/dts/armada-7040-rz-db_router.dts
index 575cffe..5cac774 100644
--- a/arch/arm/dts/armada-7040-rz-db_router.dts
+++ b/arch/arm/dts/armada-7040-rz-db_router.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_rxaui.dts b/arch/arm/dts/armada-7040-rz-db_rxaui.dts
index e43a0e1..27dd02d 100644
--- a/arch/arm/dts/armada-7040-rz-db_rxaui.dts
+++ b/arch/arm/dts/armada-7040-rz-db_rxaui.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts b/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts
index e5ef10e..4621fae 100644
--- a/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts
+++ b/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts
@@ -18,7 +18,7 @@
 
 /dts-v1/;
 
-#include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-rz.dtsi" /* include SoC device tree */
 #include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
diff --git a/arch/arm/dts/armada-70x0-rz.dtsi b/arch/arm/dts/armada-70x0-rz.dtsi
new file mode 100644
index 0000000..87cfab5
--- /dev/null
+++ b/arch/arm/dts/armada-70x0-rz.dtsi
@@ -0,0 +1,24 @@
+/*
+ * ***************************************************************************
+ * Copyright (C) 2015 Marvell International Ltd.
+ * ***************************************************************************
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation, either version 2 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * ***************************************************************************
+ */
+
+#include "apn-806-z1.dtsi"
+#include "cp110-master.dtsi"
+
+/ {
+	compatible = "marvell,armada-70x0";
+};
-- 
1.9.1

